jt-colorindexer/src/main/java/it/geosolutions/jaiext/colorindexer/Quantizer.java

```diff
@@ -88,6 +95,12 @@ public ColorIndexer buildColorIndexer(RenderedImage image) {
 
         // setup the first box, that median cut will split in parts
         List<Box> boxes = new ArrayList<Box>();
+        if (histogram.hasTransparentPixels()) {
+            BufferedImage transparentImage = new BufferedImage(1, 1, BufferedImage.TYPE_INT_ARGB);
+            transparentImage.getRaster().setPixel(0, 0, new int[] {0, 0, 0, 0});
+            PackedHistogram transparentHisto = new PackedHistogram(transparentImage, subsx, subsy);
```

Comment:
I'm wondering if we could have this "utility" PackedHistogram being statically initialized